.github/workflows/ci.yml

```diff
@@ -152,6 +152,35 @@ jobs:
           gcloud alpha storage cp "${BUILD_DIR_ARCHIVE}" "${BUILD_DIR_GCS_ARTIFACT}"
           echo "::set-output name=build-dir-gcs-artifact::${BUILD_DIR_GCS_ARTIFACT}"
 
+  build_test_all_windows:
+    needs: setup
+    if: needs.setup.outputs.should-run == 'true' && needs.setup.outputs.ci-stage == 'postsubmit'
+    runs-on: managed-windows-cpu
+    defaults:
+      run:
+        shell: bash
+    env:
+      BUILD_DIR: build-windows
+      IREE_VULKAN_DISABLE: 1
+    steps:
+      - name: "Checking out repository"
+        uses: actions/checkout@7884fcad6b5d53d10323aee724dc68d8b9096a2e # v2
+        with:
+          submodules: true
+      - name: "Setting up Python"
```

Comment:
Consider caching: https://github.com/actions/setup-python/blob/main/docs/advanced-usage.md#caching-packages

You'd need to specify the build_requirements.txt path in there as well.

Reply:
I don't think the cache is worth it for these deps.

Cold cache: https://github.com/iree-org/iree/actions/runs/3448240303/jobs/5755079803 (11s + 16s)
Warm cache: https://github.com/iree-org/iree/actions/runs/3448339897/jobs/5755276844 (6s + 13s, `Cache Size: ~15 MB (15715178 B)`)

Better to keep this file simpler and eventually migrate _all_ the deps (Python, CUDA, etc.) to Docker, IMO